### ISSUE TITLE
Unit tests fail - Ubuntu 17.10 (1 of 5)

### ISSUE THREAD

**Issue by bradh** [ngageoint/opensphere#6]
`$ npm run test` on 45f3158583cf212f10e4eb746509db19f14a68da produces unit test errors on Ubuntu 17.10 using Chrome "63.0.3239.108 (Official Build) (64-bit)".

```
Chrome 63.0.3239 (Linux 0.0.0) os.bearing should get magnetic bearings correctly FAILED
	Expected 44.22224540742933 to be close to 44.22244309187904, 8.
	Error: Expected 44.22224540742933 to be close to 44.22244309187904, 8.
	    at jasmine.Spec.<anonymous> (test/os/bearing/bearing.test.js:52:71)
	Expected 45.373682324668195 to be close to 45.37388000911791, 8.
	Error: Expected 45.373682324668195 to be close to 45.37388000911791, 8.
	    at jasmine.Spec.<anonymous> (test/os/bearing/bearing.test.js:53:67)
	Expected 112.16026829059825 to be close to 112.16177561564162, 8.
	Error: Expected 112.16026829059825 to be close to 112.16177561564162, 8.
	    at jasmine.Spec.<anonymous> (test/os/bearing/bearing.test.js:55:74)
	Expected 189.870867311387 to be close to 189.87237463643038, 8.
	Error: Expected 189.870867311387 to be close to 189.87237463643038, 8.
	    at jasmine.Spec.<anonymous> (test/os/bearing/bearing.test.js:56:70)
```

That could just be a floating point accuracy problem (32 bit / 64 bit, or compiler variations), but I wouldn't have expected it to be quite that large.

**Comment by wallw-bits** [comment]
This ended up being one of the more odd things I have ever debugged. We make use of some code derived from https://github.com/cmweiss/geomagJS (lives in `vendor/geomag`, apparently someone opted to minify it). I posted [this issue](https://github.com/cmweiss/geomagJS/issues/4) in that project earlier today after isolating the issue to that library.

The maintainer of that project recommends that we use a different file from the same project.

**Comment by wallw-bits** [comment]
Closed by #26
